tabs/src/components/app/views/RSCList.jsx

```diff
@@ -22,43 +23,63 @@ export function RSCList() {
                 rscType = "/teams/" + context?.groupId;
             } else if (context?.chatId) {
                 rscType = "/chats/" + context?.chatId;
+            } else if (context){
+                setAlert(true);
             }
 
             if (context) {
-                const rscResponse = await makeGraphCall(requestTypes.GET, [], rscType + "/permissionGrants", graphVersions.beta);
-                const rscJson = await rscResponse.json();
-                const RSCs = rscJson.value;
+                try{
```

Comment:
why was try catch added?

Reply:
in the case that the call failed

Reply:
i think the call should be checked if it returns a bad response vs. try catch but up to u

Reply:
eh i think leaving try catch is ok but not the best practice, since the call and conversion to json could both fail